syntax = "proto3";

package cta;

message Vocabularies {
  repeated Word known_words = 1;
}

message Word {
  string head_word = 1;
  string pinyin = 2;
  repeated string definitions = 3;
  // Maybe add sample sentence?
}

